salt/metalk8s/kubernetes/controller-manager/installed.sls

```diff
@@ -45,5 +41,5 @@ Create kube-controller-manager Pod manifest:
           - path: /etc/kubernetes/controller-manager.conf
             name: kubeconfig
             type: File
-    - require:
+    - onchanges:
```

Comment:
Also a requisite, no?

Reply:
So, from what I understand, `onchanges` behaves exactly like a `require` if the referenced state does not return information about changes (so only executes if the requisite returned `True`), otherwise looks at the changes to see if needs to execute. Ordering should be taken care of nicely, if that was your concern.